meerkat_abacus/config.py

```diff
@@ -116,7 +116,8 @@ def __init__(self):
         elif self.stream_data_source == "AWS_S3":
             self.get_data_from_s3 = 1
             self.s3_data_stream_interval = os.environ.get("S3_DATA_STREAM_INTERVAL", 3600)
-
+        elif self.stream_data_source == "NO_STREAMING":
```

Comment:
I'd add a final else and throw exception if unsupported STREAM_DATA_SOURCE provided:
```python
else:
  raise IncorrectArgumentError(f"STREAM_DATA_SOURCE = {self.stream_data_source} is unsupported.")
```
It may save us some headache with typo in configuration like:
```
STREAM_DATA_SOURCE: ASW_S3
```

Reply:
I agree with this.  Will make the change.

Reply:
This change has been made, can you approve @tomeksabala  or @Gulfa so we can merge?